robottelo/helpers.py

```diff
@@ -262,16 +262,24 @@ def invalid_names_list():
     ]
 
 
-def invalid_values_list():
-    """List of invalid values for input testing which consists from invalid names
-    and few empty string examples. That method should be useful for CLI and API
-    tests primarily as we can pass such parameters to a server, but not in UI.
-    As we do not verify error messages, it is recommended to use current method
-    instead 'invalid_names_list' to have better coverage and prevent redundant
-    code.
+def invalid_values_list(interface=None):
+    """List of invalid values for input testing.
 
+    This returns invalid values from from invalid_names_list() and some
+    interface (api/cli/ui) specific empty string values.
+
+    :param str interface: Interface name (one of api/cli/ui).
+    :return: Returns the invalid values list
+    :rtype: list
     """
-    return ['', ' ', '\t'] + invalid_names_list()
+    # Responsibility of the developer to call right values - api/cli/ui. If
+    # not the input will be ignored.
+    if interface not in ['api', 'cli', 'ui']:
+        interface = None
+    if interface is None or interface == 'api' or interface == 'cli':
+        return ['', ' ', '\t'] + invalid_names_list()
+    elif interface is 'ui':
+        return ['', ' '] + invalid_names_list()
```

Comment:
Would it be better to do this instead?
```
if interface is 'ui':
    return stuff
else:
    return other_stuff
```
Basically if we are treating all interfaces other than ui the same, we could simply the conditional quite a bit.